File: JavaScript/calificacion.js
let cuantasNotasVasIngresar = parseInt (prompt("Cuantas notas vas ingresar: "))
let notas
let promedio = 0

for (let index = 1; index <= cuantasNotasVasIngresar; index++) {
    notas = parseFloat(prompt(`Calificación ${index}`))
    document.write(`Calificaión ${index}: ${notas}  <br> `)
    promedio += notas / cuantasNotasVasIngresar
}   
document.write(promedio, "<br>") 

if (promedio < 3) {
    document.write("Tu calificación es F.")
}

else if (promedio < 8) {
    document.write("Tu calificación es: D.")
}

else if (promedio < 11) {
    document.write("Tu calificación es: C.")
}

else if (promedio < 15) {
    document.write("Tu calificación es: B.")
}

else {
    document.write("Tu calificación es: A.")
}

File: Python/calificacion.py
cuantasNotasVasIngresar = int(input("Cuantas notas vas a ingresar: "))
promedio = 0

for i in range (1, cuantasNotasVasIngresar + 1):
    notas = input(f"Calificación {i}: ")
    notas = float(notas)
    promedio += notas / cuantasNotasVasIngresar

print(f"Promedio final: {round(promedio)}")

if promedio < 3:
    print("Tu calificación es F.")

elif promedio < 8:
    print("Tu calificación es: D.")

elif promedio < 11:
    print("Tu calificación es: C.")

elif promedio < 15:
    print("Tu calificación es: B.")

else:
    print("Tu calificación es: A.")
    
